client/dist/styles/reviews.css

```diff
@@ -35,6 +35,13 @@
 .reviewList button {
   margin: 10px
 }
+.reviewList select {
+  border: 0px;
```

Comment:
Consider Leslie's recommendation about not hard coding pixels, but using units such as em since em scales for the display, but pixels don't.